agent/src/agent/api/routes/pipeline.py

```diff
@@ -162,10 +163,12 @@ def pipeline_status_change(pipeline_id: str):
 @pipelines.route('/pipeline-offset/<pipeline_id>', methods=['POST'])
 @needs_pipeline
 def pipeline_offset_changed(pipeline_id: str):
-    pipeline.manager.update_pipeline_offset(
-        pipeline.repository.get_by_id(pipeline_id),
-        float(request.get_json()['offset'])
-    )
+    offset_ = float(request.get_json()['offset'])
+    pipeline_ = pipeline.repository.get_by_id(pipeline_id)
+    labels_ = (pipeline_.streamsets.url, pipeline_.name, pipeline_.source.type)
+    pipeline.manager.update_pipeline_offset(pipeline_, offset_)
+    logger.info(f'offset_: {offset_}, now: {datetime.now().timestamp()}')
```

Comment:
is it very important to log this info? It might be too much data and our mechanism of enabling different log levels is not working now

Reply:
Sure not needed